doc/api/modules.md

```diff
@@ -22,9 +22,9 @@ Here are the contents of `circle.js`:
 ```js
 const { PI } = Math;
 
-exports.area = (r) => PI * r * r;
+exports.area = r => PI * r * r;
```

Comment:
Nit: not sure about other docs examples but this is inconsistent with core style where `()` is also used for a single argument.